releasenotes/notes/c-api-basis-translator-fd3b1a8d30011f2c.yaml

```diff
@@ -0,0 +1,28 @@
+---
+features_c:
+  - |
+    Add support for the :class:`.BasisTranslator` transpiler pass to the C API
+    through the function :cpp:func:`qk_transpiler_pass_standalone_basis_translator`.
+
+    Here's an example of how it should be used.
+
+    .. code-block:: C
+
+      #include <qiskit.h>
+
+      QkCircuit *circuit = qk_circuit_new(3, 0);
+      qk_circuit_gate(circuit, QkGate_CCX, (uint32_t[3]){0, 1, 2}, NULL);
+
+      // Create Target compatible with only U gates, with global props.
+      QkTarget *target = qk_target_new(3);
+      qk_target_add_instruction(target, qk_target_entry_new(QkGate_H));
+      qk_target_add_instruction(target, qk_target_entry_new(QkGate_T));
+      qk_target_add_instruction(target, qk_target_entry_new(QkGate_Tdg));
+      qk_target_add_instruction(target, qk_target_entry_new(QkGate_CX));
+
+      // Run pass
+      qk_transpiler_pass_standalone_basis_translator(circuit, 0, target);
+
+      // Free the circuit and target pointers once you're done
+      qk_circuit_free(circuit);
+      qk_target_free(circuit);
```

Comment:
```suggestion
      qk_target_free(target);
```

Reply:
Replaced in 2083432f29b7331be7002123a0c1d4b1206260b6